2682. Find the Losers of the Circular Game/solution.py

```diff
@@ -0,0 +1,29 @@
+class Solution:
+    # taken the number of players, n, and a constant, k, it returns the losers of a circular game. player 1 starts with the ball,
+    # and passes it to the person k times away from him. The same thing for 2*k. Then 3*k. And so on until the player who has 
+    # received the ball 2 times wins and the players who didn't receive the ball at all lose.
+    def circularGameLosers(self, n: int, k: int) -> List[int]:
+        # define an array of the people who will currently lose
+        losers = []
+        for i in range(n):
+            losers.append(i + 1)
+        print(losers)
```

Comment:
Can't you just use list(range())? Also, why is there still a print statement in here?